app/views/appointments/new.html.erb

```diff
@@ -11,13 +11,30 @@
         <div class="appointment-wrap animate-box" data-turbolinks="false">
           <%= form_for @appointment do |f| %>
             <%= render "shared/error", object: f.object %>
-            <div class="form-group">
-              <%= f.label :time %>
-              <%= f.text_field :time, id: "datepicker", class: "form-control", placeholder: t(".select_date"), required: true, autocomplete: :off %>
-            </div>
 
-            <div id="list-doctor">
-            </div>
+            <% if @params.nil? %>
+              <div class="form-group">
+                <%= f.label :time %>
+                <%= f.text_field :time, id: "datepicker", class: "form-control", placeholder: t(".select_date"), required: true, autocomplete: :off %>
+              </div>
+
+              <div id="list-doctor">
+                <%= render "select_doctors/doctor_select" %>
+              </div>
+            <% else %>
+              <div class="form-group">
+                <%= f.label :time %>
+                <%= f.text_field :time, id: "datepicker1", class: "form-control", placeholder: t(".select_date"), required: true, autocomplete: :off %>
+              </div>
+
+              <div id="list-doctor">
+                <%= fields_for @doctor do |f| %>
```

Comment:
fields_for có cần thiết k?
cái này chỉ in ra mỗi tên bác sĩ mà

Reply:
e sửa rồi ạ